app/scripts/modules/core/pipeline/config/stages/core/ExecutionMarkerIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import {IExecutionStageSummary} from 'core/domain/IExecutionStage';
+
+export interface IExecutionMarkerIconProps {
+  stage: IExecutionStageSummary;
+}
+
+export class ExecutionMarkerIcon extends React.Component<IExecutionMarkerIconProps, any> {
+
+  constructor(props: IExecutionMarkerIconProps) {
+    super(props);
+  }
+
+  public render() {
+    if (this.props.stage.inSuspendedExecutionWindow) {
```

Comment:
this could just be written `return this.props.stage.inSuspendedExecutionWindow ? <span className="fa fa-clock-o"/> : null;`

render() returns a `React.ReactElement< ExecutionMarkerIcon>` type, but every time I type that out, I think "why do we care?" since we never rely on the return of the render function directly. I'm happy to leave typings off of the render function...